pokedex/data/csv/pokemon.csv

```diff
@@ -1049,3 +1054,5 @@ id,identifier,species_id,height,weight,base_experience,order,is_default
 10155,necrozma-dusk,800,38,4600,306,954,0
 10156,necrozma-dawn,800,42,3500,306,955,0
 10157,necrozma-ultra,800,75,2300,339,956,0
+20000,calyrex-ice-rider,20000,24,8091,,,1
```

Comment:
I expect `calyrex-ice-rider` to have `species_id` equal to `898`. Same for the `shadow-rider`

Reply:
@Naramsim Understood, thanks. I'll make the requested changes!

Reply:
Changed to the correct `species_id` of 898 for both `shadow-rider` and `ice-rider` forms of Calyrex.